forge-dark/forge-dark-theme.scss

```diff
@@ -207,20 +213,24 @@ $node-icon-background-color-opacity: 0.05;
 $node-icon-border-color: #000;
 $node-icon-border-color-opacity: 0.1;
 
+$node-config-background: #f3f3f3;
```

Comment:
```suggestion
$node-config-background: #f1f2f9;
```

Reply:
A slight blue tinge helps it blend in better.

Thank you for pointing out the missing var